arch/arm64/boot/dts/rockchip/overlays/radxa-zero3-waveshare2-lcd.dts

```diff
@@ -0,0 +1,39 @@
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/ {
+	metadata {
+		title = "Enable Waveshare 2inch LCD";
+		compatible = "radxa,zero3";
+		category = "misc";
+		exclusive = "GPIO4_C3", "GPIO4_C2", "GPIO4_C6", "GPIO3_C1", "GPIO3_B2", "GPIO3_A2", "GPIO3_A5", "GPIO3_A6", "GPIO3_A7", "GPIO3_B4", "GPIO3_A4", "GPIO3_B3", "GPIO1_A4", "GPIO3_C3", "spi3";
```

Comment:
This for sure is more than what the devices uses. SPI bus is usually 4 wires and you have 3 extra GPIOs. Please remove unnecessary ones.